config/routes.rb

```diff
@@ -6,6 +6,15 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new] do
+      resources :comments, only: [:create]
+    end   
   end
+
```

Comment:
- [ ] There should be a route in the config/routes.rb file for creating like. Please add it.